test/spectral/mockFiles/swagger/enabled-rules.yml

```diff
@@ -166,6 +199,19 @@ definitions:
           - 2
           - a_string
           - 8
+  Pet:
+      type: object
+      description: pet object
+      properties:
+        pet_id:
+          description: pet_id
+          type: integer
+          format: int64
+          example: 10
+        name:
+          description: pet name
+          type: string
+          example: doggie
```

Comment:
This is incredibly picky but it will demonstrate that I _did_ look somewhat carefully.  I think the section I highlighted is indented one level more than it needs to be.

Reply:
Yes, nice catch!